### PR TITLE
Add X (twitter) icon to footer and update sprite.svg file

### PR DESCRIPTION
**Pull request by KellyKiiru**

### CODE REVIEW

File: src/components/Footer.tsx [hunk 1]
```diff
@@ -137,6 +137,14 @@ export default function Footer() {
               >
                 <Sprite symbol="telegram" width="18px" height="18px" />
               </a>
+              <a
+                href="https://x.com/joinmarket"
```
Comment (kristapsk) [inline src/components/Footer.tsx:141]: NACK, this account is controlled by Chris Belcher, so is basically dead currently.

Comment (KellyKiiru) [reply src/components/Footer.tsx:141]: Oh alright. Mind directing me to the right account?

Comment (kristapsk) [reply src/components/Footer.tsx:141]: There isn't one for JoinMarket currently.

But guess Jam could have one? @theborakompanioni ?

Comment (theborakompanioni) [reply src/components/Footer.tsx:141]: No! Removing all links to walled garden platforms is what should be done :wink: 
https://github.com/joinmarket-webui/jam/issues/750

**Comment by theborakompanioni** [comment]
Thanks for the effort @KellyKiiru!
Has mentioned in #750, removing the links (and icons) altogether would be great.
Could be done in this PR if you want. What do you think?

File: src/components/Footer.tsx (hunk 1 above)
Comment (KellyKiiru) [reply src/components/Footer.tsx:141]: Aaah shaks! haha I was so happy when I made the changes. Anyway, I noticed that the footer was alright and doesn't have any twitter links. We can close this issue. 

Lemme go through the issues tab and see what else I can take on

Logo

**Comment by theborakompanioni** [comment]
@KellyKiiru Are you still working on this? :pray:

**Comment by KellyKiiru** [comment]
Hey @theborakompanioni. No.

**Comment by theborakompanioni** [comment]
> Hey @theborakompanioni. No.

Thanks for the quick reply! Closing the PR now!
Please feel free to pick it up again (or any other issue that you feel suitable)!
Thank you :orange_heart:

**Pull request closed without merging**